docs/usage.md

```diff
@@ -386,12 +379,20 @@ See `--genome1` above for more details.
 
 ### `--fasta3`
 
-Path to canidate organism 3 genome fasta file (must be surrounded with quotes). Must be provided if `--genome3` is not provided
+Reference genome3 can be specified by using the full path to the genome fasta file. Must be provided if `--genome3` is not provided.
```

Comment:
Should go before `--genome3`, as you ddid above already.

Reply:
Updated with 180805b